### PR TITLE
Feature/reviews

### PR DESCRIPTION
**Pull request by illja-popovics**
Hi! I have added id for the section for the navigation

### CODE REVIEW

**Comment by illja-popovics** [comment]
Hi guys, I have added an ID to my section and adjusted the --desktop-container-max-width variable. Please test if everything is OK.  I didn't notice any issues. This way the content width is 1024, as our mentor approved.

File: src/css/base.css [hunk 1]
```diff
@@ -15,7 +15,7 @@
   --transition: 250ms cubic-bezier(0.4, 0, 0.2, 1);
 
   --tablet-container-max-width: 704px;
-  --desktop-container-max-width: 1024px;
+  --desktop-container-max-width: 1088px;
```
Comment (kazamov) [inline src/css/base.css:18]: I have checked the design, the max width for the container on desktop is 1024px, please revert

**Review by kazamov — CHANGES_REQUESTED** [review]

File: src/css/base.css (hunk 1 above)
Comment (illja-popovics) [reply src/css/base.css:18]: Hi Zakir, please check the content width on the current build, it is less than on the design: 
image

Comment (kazamov) [reply src/css/base.css:18]: You are right, approve

**Review by kazamov — APPROVED** [review]

**Pull request merged**